packages/help-center/src/components/help-center-header.tsx

```diff
@@ -50,8 +50,16 @@ const HelpCenterMobileHeader: React.FC< Header > = ( {
 	return (
 		<CardHeader className={ classNames }>
 			<Flex>
-				<p className="help-center-header__text">
-					<span className="help-center-header__icon">{ headerText }</span>
+				<p
+					style={ {
+						maxWidth: 300,
+						fontSize: 14,
+						fontWeight: 500,
+						display: 'flex',
+						alignItems: 'center',
+					} }
+				>
+					<span className="help-center-header__title">{ headerText }</span>
```

Comment:
Why did you remove the class and add again the inline styles? I see that `help-center-header__text` is still in the scss file, and I'm not sure but could be [losing some styles](https://github.com/Automattic/wp-calypso/pull/63639/files#diff-106247b3ddab08d2998bd029a32a570711ab72f884cc01410803133094e59406R40-R44)

Reply:
Hmm I thought you added that. I think I may have gotten some branches crossed while working on this. I will remove it. Nice catch! 🎣

Reply:
Well I added the `help-center-header__text` class and removed the inline styles 😄 I think we should not have the inline styles but the class I introduces, wdyt?

Reply:
Totally agree. I fixed it and tested. Everything is working great still. Also updated this `inline-help__resource-cell a, still maintains the focus after clicking (the blue border)`